packages-exp/auth-exp/test/integration/webdriver/static/index.js

```diff
@@ -18,12 +18,14 @@
 import * as redirect from './redirect';
 import * as anonymous from './anonymous';
 import * as core from './core';
+import * as persistence from './persistence';
 import { initializeApp } from '@firebase/app-exp';
 import { getAuth, useAuthEmulator } from '@firebase/auth-exp';
 
 window.core = core;
 window.anonymous = anonymous;
 window.redirect = redirect;
+window.persistence = persistence;
```

Comment:
I haven't merged it yet but I've changed all of these so that they no longer set the whole module and instead set an object (so that the individual modules can namespace):

`window.persistence = {...persistence};`

Reply:
Done. Will clean this up in a follow up PR